### ISSUE TITLE
What fork is actively maintained?

### ISSUE THREAD

**Issue by Lewik** [helun/Ektorp#300]
What fork is actively maintained?

**Comment by bashizip** [comment]
I'm trying  http://github.com/cloudant/java-cloudant witch has also support for local CouchDB instances.

**Comment by Lewik** [comment]
Is Ektorp abandoned?

**Comment by Lewik** [comment]
I found https://github.com/triplem/Ektorp/tree/v1.6
@triplem , I can't contact with you. How to use your fork? Is it in maven? What plans?

**Comment by triplem** [comment]
Yeah, well, the latest incarnation is not in maven central. You could build it yourself though. 
Take a look at https://github.com/triplem/Ektorp/tree/v1.6?files=1.
The plan is to add mor features for Couchdb 2.3 like Mango.

**Comment by Lewik** [comment]
@triplem I used jitpack to use your fork. All works btw.
I didn't understand `?files=1` Without this result is the same, what do you want to show to me?
